pydantic/main.py

```diff
@@ -671,30 +671,6 @@ def model_validate_strings(
         __tracebackhide__ = True
         return cls.__pydantic_validator__.validate_strings(obj, strict=strict, context=context)
 
-    @classmethod
-    def __get_pydantic_core_schema__(cls, source: type[BaseModel], handler: GetCoreSchemaHandler, /) -> CoreSchema:
```

Comment:
I dont think this is a good idea. This is definitely being relied on. Its not internal and also documented for customizing the schema.

Reply:
To be clear, this doesn't remove support for `__get_pydantic_core_schema__`. This is still handled by the added `_generate_schema_from_get_schema_method` method.

Users are still free to implement `__get_pydantic_core_schema__` on Pydantic models, however we provide no guarantees (and never did, see the _breaking changes concerns_ section in my original post) it will work flawlessly.

Reply:
Ah interesting. Will give this another review then. Didn't realize users could still implement support. Nice!